### PR TITLE
update service view

### PR DESCRIPTION
**Pull request by YitongFeng**
## Update:
- Disable Label click action
- Table row shows in one line
- Add link to nodename
- Add link to container log
image

### CODE REVIEW

File: webportal/src/app/cluster-view/services/service-table.component.ejs [hunk 1]
```diff
@@ -8,23 +9,28 @@
...
+              <a class="btn btn-success" href="<%= k8sUri + '/#!/log/' + pod.namespace + '/' + pod.name + '/?namespace=' + pod.namespace  %>"> 
+              <%= data[i].podList[j].metadata.name %> </a>  
             <% } else{ %>
-              <button type="button" class="btn btn-danger"> <%= data[i].podList[j].metadata.name %> </button>
+              <a class="btn btn-danger" href="<%= k8sUri + '/#!/log/' + pod.namespace + '/' + pod.name + '/?namespace=' + pod.namespace  %>"> <%= data[i].podList[j].metadata.name %> </a>
             <% } %>      
           <% } %>
         </td>
-      </tr>
+      </tr>
```
Comment (abuccts) [inline webportal/src/app/cluster-view/services/service-table.component.ejs]: trailing whitespace

File: webportal/config/webportal.config.js [hunk 2]
```diff
@@ -27,7 +27,8 @@ let config = {
   restServerUri: process.env.REST_SERVER_URI,
   grafanaUri: process.env.GRAFANA_URI,
   k8sDashboardUri: process.env.K8S_DASHBOARD_URI,
-  k8sApiServerUri: process.env.K8S_API_SERVER_URI
+  k8sApiServerUri: process.env.K8S_API_SERVER_URI,
+  exportPort: process.env.EXPORT_PORT
```
Comment (abuccts) [inline webportal/config/webportal.config.js]: change a variable name? `exportPort` is confusing.

File: webportal/src/app/cluster-view/services/service-table.component.ejs (hunk 1 above)
Comment (YitongFeng) [reply webportal/src/app/cluster-view/services/service-table.component.ejs]: Solved.

File: webportal/config/webportal.config.js (hunk 2 above)
Comment (YitongFeng) [reply webportal/config/webportal.config.js]: Changed to exporterPort

**Pull request merged**